nodeup/pkg/model/kubelet_test.go

```diff
@@ -182,6 +182,24 @@ func Test_RunKubeletBuilder(t *testing.T) {
 	}
 	context.AddTask(fileTask)
 
+	{
+		task, err := builder.buildManifestDirectory(kubeletConfig)
+		if err != nil {
+			t.Fatalf("error from KubeletBuilder buildManifestDirectory: %v", err)
+			return
+		}
+		context.AddTask(task)
+	}
+
+	{
+		task := builder.buildSystemdService()
+		if err != nil {
+			t.Fatalf("error from KubeletBuilder buildSystemdService: %v", err)
+			return
+		}
+		context.AddTask(task)
+	}
+
```

Comment:
Hmmm... I'm trying to remember why we don't just call `Build`...  this change is right, but I'm just hoping you might know why we don't call Build?

Reply:
(And I guess this is why we split out the `buildManifestDirectory` function ... which otherwise it would feel more natural to inline)

Reply:
@justinsb, for me, I followed the pattern of breaking it out into a separate function call b/c the test (kubelet_test.go) was imitating the internal calls inside the build method, and needed access to what build was actually calling.  It felt weird when I saw that, but I limited my scope to just fix the issue, and not how kubelet's build method is tested.